### DISCUSSION TITLE
For validators, what is the default `mode`?

### DISCUSSION THREAD

**Discussion by ecs-jnguyen** [pydantic/pydantic#7913] [Question] [answered]
I'm going through the documentation for the validators and I'm a bit unclear on what the default value for the `field_validator` is. Is it `after`?

Reference:
- https://docs.pydantic.dev/latest/concepts/validators/#before-after-wrap-and-plain-validators

**Comment by sydney-runkle** [comment, marked as answer]
Yep, it's after! You can read more about that in our API docs: https://docs.pydantic.dev/latest/api/functional_validators/#pydantic.functional_validators.field_validator
